airflow/configuration.py

```diff
@@ -41,6 +41,22 @@ def expand_env_var(env_var):
         else:
             env_var = interpolated
 
+def run_bash_command(command):
+    """
+    Runs bash command and returns the derived value.
+    """    
+    if not command:
+        return command
+
+    import subprocess
```

Comment:
imports goes at module top, except in unusual cases, `utils.py` might have it already as you move it